tests/sentry/api/test_data_secrecy.py

```diff
@@ -34,3 +34,41 @@ def test_non_member_no_access(self):
 
     def test_member_has_access(self):
         self.get_success_response(self.organization.slug)
+
+
+class DataSecrecyV2TestCase(APITestCase):
```

Comment:
should we test that if an org has data secrecy and you're staff, you can access org endpoints used in admin, but if you're superuser at all, you can't?